.github/workflows/release.yaml

```diff
@@ -59,10 +59,46 @@ jobs:
           NPM_PACKAGE_ROOT: "npm"
           SKIP_DOCKER_PUBLISH: true
 
+  docs:
+    name: Update documentation
+    needs:
+      - release
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Generate token
+        id: generate_token
+        uses: tibdex/github-app-token@v2
+        with:
+          app_id: ${{ secrets.OS_GITHUB_APP_ID }}
```

Comment:
I'm getting this warning in VS Code. I think it's OK, as I'm doing the same thing the release job is doing.

![CleanShot 2024-09-09 at 15 17 52](https://github.com/user-attachments/assets/52996273-f87c-4b71-a1bf-86e97a9d88ef)